### PR TITLE
git-game 1.3

### PR DESCRIPTION
**Pull request by coffee-dan**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
fatal: 'game' appears to be a git command, but we were not
```

**Comment by coffee-dan** [comment]
Currently investigating the CI fail.

**Comment by coffee-dan** [comment]
Some progress. It looks like there is no ruby available in the `homebrew/ubuntu22.04:latest`. I am now looking into how to specify dependencies in the formula definition.

For what it is worth, `brew test git-game` also fails for the previous version.

File: Formula/git-game.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class GitGame < Formula
   desc "Game for git to guess who made which commit"
   homepage "https://github.com/jsomers/git-game"
-  url "https://github.com/jsomers/git-game/archive/1.2.tar.gz"
-  sha256 "d893b2c813388754c16d867cf37726cd7e73c9ccd316735aac43bf1cd3ab1412"
+  url "https://github.com/coffee-dan/git-game/archive/1.3.tar.gz"
```
Comment (carlocab) [inline Formula/git-game.rb:4]: We should probably check with @jsomers that they're happy for you to take over the project before switching this to your repo.

**Review by carlocab — CHANGES_REQUESTED** [review]

File: Formula/git-game.rb (hunk 1 above)
Comment (coffee-dan) [reply Formula/git-game.rb:4]: Absolutely! Thank you for pointing this out. 

I setup this PR to get ahead of CI failures. I am coordinating with @jsomers to make a 1.3 release on the official git-game repo and _that_ will be the URL for that gets used.

File: Formula/git-game.rb [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
   license "MIT"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "c700a65b591b1e95ab45bcd0349a75aeb8fdc9915480c7323a34830cd27075fd"
   end
 
+  depends_on "ruby"
+
   def install
     bin.install "git-game"
```
Comment (carlocab) [inline Formula/git-game.rb:15]: The shebang here probably needs to be `/usr/bin/env ruby` instead of `/usr/bin/ruby`.

Comment (coffee-dan) [reply Formula/git-game.rb:15]: Thank you so much! I will test that out now.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request merged**